ceph/rbd/workflows/cluster_operations.py

```diff
@@ -0,0 +1,172 @@
+import re
+import time
+from importlib import import_module
+
+from ceph.rados import utils
+from tests.rados.rados_test_util import get_device_path, wait_for_device
+from tests.rados.stretch_cluster import wait_for_clean_pg_sets
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def restart_ceph_target(admin_node):
+    """
+    Restart the Ceph target service.
+
+    Args:
+        admin_node (Node): The cephadm admin node.
+
+    Raises:
+        Exception: If the Ceph target service restart fails.
+    """
+    log.info("Restarting Ceph target service")
+    try:
+        cmd = "systemctl restart ceph.target"
+        admin_node.exec_command(cmd=cmd, sudo=True)
+    except Exception as err:
+        log.error(err)
+        raise Exception("Failed to restart ceph traget service")
+
+
+def restart_osd(client_node):
+    """
+    Method to restart ceph osd service.
+
+    Args:
+        client_node (Node): The ceph client node.
+
+    Raises:
+        Exception: If Ceph health check fails.
+    """
+    log.info("Restarting Ceph osd service")
+    try:
+        cmd = "ceph orch ls osd"
+        out, err = client_node.exec_command(cmd=cmd, output=True)
```

Comment:
exec_command method does not use the input output=True like i mentioned during our discussions, it returns output by default if there is any, so please remove this.

Reply:
done